multiversx/mx-chain-go
# New p2p network messenger for the light clients and suppliers.

## Reasoning behind the pull request
- Light clients will connect on this additional network messenger.
  
## Proposed changes
- Similar to the full archive network, another one has been added for the light client suppliers to connect to.

## Testing procedure
- Start a seednode with mvx-light-client protocol ID.
- Start a node with the InitialPeerList in lightClientP2P.toml set from the seednode you just started.
- Connection should work, add more nodes if needed. 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
